### PR TITLE
Fix deepsource errors.

### PR DESCRIPTION
**Pull request by arijitAD**
## Changes

- Fix deepsource errors.
- https://github.com/ChainSafe/gossamer/pull/1780

## Tests

```
TestBlockAnnounceHandshakeSize
```

## Issues

-

## Primary Reviewer 

- @EclesioMeloJunior

### CODE REVIEW

**Review by EclesioMeloJunior — APPROVED** [review]
lgmt

File: dot/network/host.go [hunk 1]
```diff
@@ -225,18 +225,8 @@ func (h *host) close() error {
 }
 
 // registerStreamHandler registers the stream handler, appending the given sub-protocol to the main protocol ID
```
Comment (noot) [inline dot/network/host.go:227]: update comment

File: dot/network/host.go [hunk 2]
```diff
@@ -251,16 +241,18 @@ func (h *host) connect(p peer.AddrInfo) (err error) {
 // bootstrap connects the host to the configured bootnodes
 func (h *host) bootstrap() {
 	failed := 0
-	all := append(h.bootnodes, h.persistentPeers...)
-	for _, addrInfo := range all {
+	var allNodes []peer.AddrInfo
+	allNodes = append(allNodes, h.bootnodes...)
+	allNodes = append(allNodes, h.persistentPeers...)
+	for _, addrInfo := range allNodes {
```
Comment (noot) [inline dot/network/host.go:247]: did deepsource complain about this? just curious what the issue was

**Review by noot — APPROVED** [review]
looks good!

File: dot/network/host.go (hunk 2 above)
Comment (arijitAD) [reply dot/network/host.go:247]: Yes https://deepsource.io/gh/ChainSafe/gossamer/run/7db3c103-c088-4fb5-904e-bad5ef591ab6/go/CRT-D0001
```
possibly assigns to a wrong variable
```

File: dot/network/host.go (hunk 1 above)
Comment (arijitAD) [reply dot/network/host.go]: Done

**Pull request merged**